cmd/clusterctl/cmd/upgrade_plan.go

```diff
@@ -67,21 +67,32 @@ func runUpgradePlan() error {
 		return err
 	}
 
-	upgradePlans, err := c.PlanUpgrade(client.PlanUpgradeOptions{
+	certManUpgradePlan, err := c.PlanCertManagerUpgrade(client.PlanUpgradeOptions{
 		Kubeconfig: client.Kubeconfig{Path: up.kubeconfig, Context: up.kubeconfigContext},
 	})
 	if err != nil {
 		return err
 	}
+	if certManUpgradePlan.ShouldUpgrade {
```

Comment:
Might be we should print a message also in case cert-manager is already up do date (so we are explicit about it)